docker/data-science/notebooks/ch15/0.setup.ipynb

```diff
@@ -0,0 +1,64 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ltr.download import download, extract_tgz\n",
+    "from ltr.helpers.movies import indexable_movies\n",
+    "import tarfile\n",
+    "import json\n",
+    "\n",
+    "dataset = [\"https://github.com/ai-powered-search/tmdb/raw/main/judgments.tgz\", \n",
```

Comment:
Should git pull (like earlier chapters) instead of GET over https. Github sometimes rate limits IPs that do this (I've had this cause my notebooks to break recently). I think we do this also in ch11 setup and should convert to git pull.

Can push as is and adjust later; just noting.

Reply:
Ch15 debug code has been deleted, but I've gone ahead and updated ch11